sites/feedback/feedback-docs-en.html

```diff
@@ -0,0 +1,245 @@
+										&lt;input name="problem6" id="problem6a" type="checkbox" value="Yes" /&gt;GC Key access
+									&lt;/label&gt;
+									&lt;input name="problem6" type="hidden" value="" /&gt;
+								&lt;/li&gt;
+								&lt;li class="checkbox"&gt;
+									&lt;label for="problem7"&gt;
+										&lt;input name="problem7" id="problem7" type="checkbox" value="Yes" /&gt;SecureKey Concierge (Banking Credential) access
+									&lt;/label&gt;
+									&lt;input name="problem7" type="hidden" value="" /&gt;
+								&lt;/li&gt;
+								&lt;li class="checkbox"&gt;
+									&lt;label for="problem8"&gt;
+										&lt;input name="problem8" id="problem8" type="checkbox" value="Yes" /&gt;Personal Access Code (PAC) problems or EI Access Code (AC) problems
+									&lt;/label&gt;
+									&lt;input name="problem8" type="hidden" value="" /&gt;
+								&lt;/li&gt;
+								&lt;li class="checkbox"&gt;
+									&lt;label for="problem9"&gt;
+										&lt;input name="problem9" id="problem9" type="checkbox" value="Yes" /&gt;Social Insurance Number (SIN) validation problems
+									&lt;/label&gt;
+									&lt;input name="problem9" type="hidden" value="" /&gt;
+								&lt;/li&gt;
+								&lt;li class="checkbox"&gt;
+									&lt;label for="problem10"&gt;
+										&lt;input name="problem10" id="problem10" type="checkbox" value="Yes" /&gt;Other login error not in this list
+									&lt;/label&gt;
+									&lt;input name="problem10" type="hidden" value="" /&gt;
+								&lt;/li&gt;
+							&lt;/ul&gt;
+						</ins>
+						&lt;div class="checkbox"&gt;
+							&lt;label for="gc-rprt-prblm-f4"&gt;
+								&lt;input type="checkbox" name="problem4" value="Yes" id="gc-rprt-prblm-f4" /&gt;Information is outdated or wrong
+							&lt;/label&gt;
+							&lt;input name="problem4" type="hidden" value="No" /&gt;
+						&lt;/div&gt;
+						&lt;div class="checkbox"&gt;
+							&lt;label for="gc-rprt-prblm-f5"&gt;
+								&lt;input type="checkbox" name="problem5" value="Yes" id="gc-rprt-prblm-f5" /&gt;I can't find what I'm looking for
+							&lt;/label&gt;
+							&lt;input name="problem5" type="hidden" value="No" /&gt;
+						&lt;/div&gt;
+						&lt;div class="checkbox"&gt;
+							&lt;label for="gc-rprt-prblm-f6"&gt;
+								&lt;input type="checkbox" name="problem6" value="Yes" id="gc-rprt-prblm-f6" /&gt;Other issue not in this list
+							&lt;/label&gt;
+							&lt;input name="problem6" type="hidden" value="No" /&gt;
+						&lt;/div&gt;
+					&lt;/fieldset&gt;
+					&lt;button type="submit" class="btn btn-primary wb-toggle" data-toggle='{"stateOff": "hide", "stateOn": "show", "selector": ".gc-rprt-prblm-tggl"}'&gt;Submit&lt;/button&gt;
+				&lt;/form&gt;
+			</code>
+		</pre>
+	</div>
+	<div class="col-md-6">
+		<h6>Before version 2.0</h6>
+		<pre>
+			<code>
+				&lt;form action="#"&gt;
+					&lt;fieldset&gt;
+						&lt;legend&gt;&lt;span class="field-name"&gt;Please select all that apply:&lt;/span&gt;&lt;/legend&gt;
+						&lt;div class="checkbox"&gt;
```

Comment:
Use the `<del>` to identify the section that was removed or fully renamed

Reply:
Changed. Thanks